### ISSUE TITLE
Problem in running ImageNet demo

### ISSUE THREAD

**Issue by avisekiit** [JiahuiYu/generative_inpainting#343]
Hello,
Thanks for the awesome repo. I was trying to run the demo for ImageNet; however I keep getting some dimension mismatch error. I am using the DeepFillv1;

Using the pre-trained Imagenet, we get conv weights mismatch error (refer "error1.png"). Model is expecting weights of dimension [5, 5, 5, 48], which means the number of channels in layer 2 should be 48, but it is 32 according to the dimension of weights. Please refer "dims1.png", it shows dimensions of conv filters in the model and the dimensions of pretrained weights for all the layers.

There is a parameter named "cnum" in the model file(refer "model1.png"), it was initialized to 48. It defines the number of output layers of conv1. I changed it to 32 to resolve the mismatch.  Please refer to "dims2.png", left one is the dimension of the model and the right one is the dimension of pretrained weights. For the model weights, we can see that weights are of dimensions:
[ker, ker, in_dim, out_dim]
[5, 5, 5, 32]
[3, 3, 16, 64]
[3, 3, 32, 64]
[3, 3, 32, 128] and so on.
We can see that the input dimension of the next layer is half the output dimension of the previous layer, which is not the case for pretrained weights.
If we see the convolution function(refer "get_conv.png"), we can see that in the last 4 lines, it splits the output into two parts along the 3rd axis, then it returns their product, hence it halves the output dimension.

![get_conv](https://user-images.githubusercontent.com/12903678/68495569-82148200-0276-11ea-83f9-491eafd2f4b0.png)
![model1](https://user-images.githubusercontent.com/12903678/68495570-82ad1880-0276-11ea-8f2a-b2e0e107668d.png)
![dims2](https://user-images.githubusercontent.com/12903678/68495574-82ad1880-0276-11ea-89be-a58d00b1dfed.png)
![dims1](https://user-images.githubusercontent.com/12903678/68495575-82ad1880-0276-11ea-8361-383e96c967c4.png)
![error1](https://user-images.githubusercontent.com/12903678/68495576-8345af00-0276-11ea-9446-b177f29c2727.png)

**Comment by JiahuiYu** [comment]
@avisekiit For DeepFill v1, please check out branch v1.0.0, instead of master branch.

https://github.com/JiahuiYu/generative_inpainting/blob/v1.0.0/inpaint_model.py#L45
